tests/integration/utils.py

```diff
@@ -240,12 +247,15 @@ def fetch_deps_and_check_output(
         output_json = _load_json(output_folder.joinpath("output.json"))
         expected_output_json = _load_json(test_data_dir.joinpath(test_case, "output.json"))
 
-        if "project_files" in expected_output_json:
-            _set_tmpdir_path(expected_output_json["project_files"], tmp_path)
+        output_json["project_files"] = expected_output_json["project_files"]
```

Comment:
This would mean our tests don't check project_files

I think we want something like this:

output_json:
```json
{
    "project_files": [
        {
            "abspath": "/tmp/pytest-of-acmiel/pytest-6/test_e2e_pip_pip_e2e_test_0/pip_e2e_test-source/requirements.txt",
            "template": "...",
        }
    ]
}
```
replace abspath:
```python
test_case_tmpdir = Path("/tmp/pytest-of-acmiel/pytest-6/test_e2e_pip_pip_e2e_test_0")
_replace_abspath_with_placeholder(output_json, test_case_tmpdir)
```
result (to save in test data dir):
```json
{
  "project_files": [
    {
      "abspath": "${test_case_tmpdir}/pip_e2e_test-source/requirements.txt",
      "template": "..."
    }
  ]
}
```

Reply:
Already resolved :+1: